annotation-metadata/README.md

```diff
@@ -1,19 +1,26 @@
-# Draft Annotation Metadata Standards for EM and XRM Connectomics Community
-In this folder, you will find
-* A draft metadata description: [schema](annotation-metadata-draft.plantuml), [diagram](annotation-metadata-draft.png), [description](annotation-metadata-draft.md)
+## Annotation Metadata Schema
+
+# Overview
+The annotation-metadata directory contains the version-controlled development of annotation metadata schemas. Each schema version is designed to improve and standardize the neuroanatomical annotation process.
+
+# Contents
+* PlantUML Diagrams: These files define the structure of the annotation metadata schema across different versions (1.0, 1.1, and 1.2). These versions are evolving and will change as a result of community input. 
+* PNG Diagrams: Visual representations of the UML diagrams in PNG format for viewing.
+* Connectome Annotation Versioning Engine (CAVE) Implementation: The 1.2 version of the schema has been adapted to work with the CAVE framework, in effort to adapt community created platforms. 
+* Neuroanatomy Schema: Outlined background for the corresponding neuroanatomy structure. 
+* Required Field Names: A markdown file listing the required fields in the metadata schema.
+
+# Files
+* annotation-metadata-version-1.0.plantuml
+* annotation-metadata-version-1.0.png
+* annotation-metadata-version-1.1.plantuml
+* annotation-metadata-version-1.1.png
+* CAVE-implemented-annotation-metadata-version-1.2.plantuml
+* CAVE-implemented-annotation-metadata-version-1.2.png
+* RAMONAnnotation.java
+* neuroanatomy-schema.pdf
+* required-field-names.md
+
+# Usage
+To view the UML diagrams, you will need a PlantUML viewer or editor. The PNG files can be viewed with any image viewer. Please refer to the neuroanatomy-schema.pdf and required-field-names.md for domain-specific elements. 
```

Comment:
FWIW and for the future -- you might like https://github.com/mermaid-js/mermaid so you could have source in text etc